CHANGELOG.rst

```diff
@@ -16,6 +16,8 @@ User-Facing Changes
 * **Core Service**: Fix a crash when migrating to v10 metadata
   (`#3359 <https://github.com/SwissDataScienceCenter/renku-python/pull/3359>`__).
 * **Knowledge Graph**: Expand the Knowledge Graph Project Activation page to list all projects where the user is a member, not only owned projects.
+* **Sessions**: Properly filter sessions by project name 
+  (`#1423 <https://github.com/SwissDataScienceCenter/renku-notebooks/pull/1423>`_).
```

Comment:
Is this a user-facing change of an internal one? (I am not sure what you mean by this)

Reply:
Ok I reworded as we discussed offline.